apps/evans/pages/index.tsx

```diff
@@ -30,7 +30,7 @@ export const getStaticProps: GetStaticProps<Props> = async () => {
     newsFeedItems: getNewsFeedItems(newsFeedItems),
     numTotalBooks: totalPublished.books[LANG],
   }));
-  if (process.env.VERCEc_ENV === `production`) {
+  if (process.env.VERCEL_ENV === `production`) {
```

Comment:
lol, this typo means we haven't been sending updated search data to algolia for a while. 🤦‍♂️